add-ons/hyper-sorter/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var _ = require('object-iterators');
+
+function Hypersorter(grid, objects) {
+    this.grid = grid;
+    objects = objects || {};
+
+    mixInTo('Hypergrid', grid, require('./mix-ins/grid'));
+    mixInTo('Behavior', grid.behavior, require('./mix-ins/behavior'));
+    mixInTo('Column', grid.behavior, require('./mix-ins/column'));
```

Comment:
this won't work. following will work so long as setData has been called and there is at least one column:
```javascript
mixInTo('Column', grid.behavior.allColumns[0], require('./mix-ins/column'));
```
alternatively, if we have access to the `Column` constructor, perhaps by adding to `src/behaviors/index.js`:
```javascript
objects.Column = Object.getPrototypeOf(grid).constructor.behaviors.Column;
mixInTo('Column', undefined, require('./mix-ins/column'));
```